packages/dev/docs/src/client.js

```diff
@@ -19,7 +19,10 @@ import ShowMenu from '@spectrum-icons/workflow/ShowMenu';
 import {ThemeSwitcher} from './ThemeSwitcher';
 import {watchModals} from '@react-aria/aria-modal-polyfill';
 
-window.addEventListener('load', () => listen());
+if (process.env.NODE_ENV === 'production') {
```

Comment:
This does seem like a feature for production, but wondering if the upgrade caused this or you thought this would be useful while working on parcel upgrade?

Reply:
Yeah, originally I had enabled Parcel's lazy mode which only builds pages that you actually load in the browser, and the preloading was causing us to build more than necessary. I took out lazy, but this still seems like a production only feature anyway so I left it.